docs/benefits_of_contextualized_ML.ipynb

```diff
@@ -0,0 +1,838 @@
+{
```

Comment:
<p>These errors look much nicer than the ones before. Why?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/cnellington/Contextualized/pull/213/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='17'/>